### DISCUSSION TITLE
Can Fooocus be used to enhance the quality of existing photos?

### DISCUSSION THREAD

**Discussion by llagerlof** [lllyasviel/Fooocus#1836] [Q&A] [answered]
I have some old photos, a bit blurred, a bit grained, that I want to improve quality. Is it possible to do this using Fooocus? I tried some configurations and made some tests, but no success.

**Comment by JT-Gresham** [comment, marked as answer]
I've had some luck with some old photos using the upscale feature.  After I upscaled the image by 2x, I shrunk it back down using GIMP....then repeated the process.  This worked for several photos, but not all.  It's worth a try, I'd say.

Let us know how it works out for you.

**Reply by Amit30swgoh** [reply]
Gimp?

**Reply by llagerlof** [reply]
@JT-Gresham Did you shrink it back down to 50%, returning the photo to its original size?

**Reply by JT-Gresham** [reply]
Yes, I shrunk it back down to original size.  You might try to upscale it more than once depending on what size you actually want the finished image to be.  I think the AI upscales a bit differently than standard resize methods used by image editing software (like GIMP <--- a free and open source imaging software comparable to Photoshop) so the upscaling quality is better so, in theory, you should be able to achieve better results by using Fooocus to upscale, then use GIMP or similar to shrink it back down to restore the best possible quality.  Hope that helps.

**Comment by mashb1t** [comment]
You may also have a look at https://github.com/lllyasviel/Fooocus/issues/1490 where we restored a photo of the grandpa of a community member!

**Reply by Amit30swgoh** [reply]
My grandpa :)

**Reply by Amit30swgoh** [reply]
Does <lora nameofmodel:1.1> works ?
If i didnt put the LoRA model on yhe right bar it diidnt

**Reply by jeffersonrodrigo** [reply]
I saw the result here and it looks great 😍😍 — it’s exactly what I was looking for. I’ll try to read and understand the processes used. It would be nice to compile it into some kind of ‘how-to’.
